core/templates/pages/admin-page/misc-tab/admin-misc-tab.directive.html

```diff
@@ -1,129 +1,131 @@
-<md-card class="oppia-page-card oppia-long-text" style="max-width: 700px">
-  <h3>Topic Similarities</h3>
-  <div class="oppia-topic-similarities-container">
-    Upload a csv file:<br>
-    The first line of data should be a list of topic names. The next lines should be a symmetric adjacency matrix of similarities, which are values between 0 and 1.<br>
-    <input type="file" id="topicSimilaritiesFile">
-    <button ng-click="$ctrl.uploadTopicSimilaritiesFile()">Upload</button>
-    <button ng-click="$ctrl.downloadTopicSimilaritiesFile()">Download current topic similarities</button>
-  </div>
-</md-card>
+<div class="oppia-admin-misc-tab-container">
```

Comment:
ANy specific reason for using extra div container? (For finding elements can we use the directive/tag-name?)